src/main/java/moomoo/task/Ui.java

```diff
@@ -71,7 +74,7 @@ public void showGoodbye() {
      * @return Message of the MooMooException
      */
     public String printException(MooMooException e) {
-        this.output = e.getMessage();
+        this.output = ANSI_RED + e.getMessage();
```

Comment:
You need to ANSI_RESET the colour. If not, all your outputs afterwards will all be in red. Refer to josh's MainDisplay to get the colour for ANSI_RESET. Then you wrap your e.getMessage() text with ANSI_RED and ANSI_RESET. The reset colour part should be added to the end of the String. Can refer to my NotificationsCommand part to see how i reset the colour.